.circleci/config.yml

```diff
@@ -28,49 +24,41 @@ setup-gpg: &setup-gpg
     command: |
       echo -e "$GPG_SECRET_KEY" | gpg --import --no-tty --batch --yes
 
-version: 2
 jobs:
   build:
     <<: *job-defaults
     docker:
-      # specify the version you desire here
-      - image: circleci/openjdk:8-jdk
+      - image: cimg/openjdk:8.0
```

Comment:
not sure why these images. Why not `maven:3.9-eclipse-temurin-17` ?

Reply:
Are these CircleCI images?

Reply:
Yes, they are circleci images, I tried with maven image as well. Please can see my commits. 
It only worked with these images [https://circleci.com/developer/images/image/cimg/openjdk](https://circleci.com/developer/images/image/cimg/openjdk)